packages/berlin/src/pages/Register.tsx

```diff
@@ -249,9 +273,18 @@ function TextInput(props: {
             return v.error.errors[0].message;
           },
         })}
-        errors={[props.errors?.[props.id]?.message ?? '']}
         disabled={props.disabled}
+        onChange={handleInputChange}
       />
+      {props.errors?.[props.id] ? (
+        <Error>{props.errors?.[props.id]?.message}</Error>
+      ) : (
+        props.characterLimit > 0 && (
```

Comment:
There is a small height jump went the char count is rendered, as the Error seems to have 16px of height and the <p> 21px. Solution here would be to modify the `line-height` to match the Error one, with is `1rem` = `16px`

Or better yet, a custom Typography component for this character count, as we might need it in the future at other places :)

Reply:
Implemented typography component and adjusted font size.